### DISCUSSION TITLE
VSF TR-11 has been published on vsf.tv -- Comments and Discussion Here!

### DISCUSSION THREAD

**Discussion by jmailhot** [vsf-tv/gccg-api#15] [General]
The VSF GCCG working group's draft recommendation has been published at https://vsf.tv/download/technical_recommendations/VSF_TR-11_2024-02-21-draft.pdf
We look forward to discussion of how its being used, and how it could be improved.

**Comment by mikeov1** [comment]
First comment - in the COT definition it states:
 "This timestamp may be accompanied by a discontinuity indicator which alerts when there is a discontinuous change to the timestamp at a given Media Element."   

but a timestamp is a single value so cannot be discontinuous.  I suggest that the word "sequence" be inserted so it reads:

 "This timestamp may be accompanied by a discontinuity indicator which alerts when there is a discontinuous change to the timestamp sequence at a given Media Element."


Second comment
Section 7.2 states ".... withing 50ppm (parts per million) of traceable time"  but 50ppm is a frequency spec, not a time spec.  Is that really a valid way to spec that?.   Then in section 10, the 50ppm limit is reiterated and a 100ms offset from traceable time is added.    If the timebase frequency is off by 50ppm, then the phase will drift.  50ppm is 50us of drift per second, so 100ms in 2000 seconds.    So even if the timestamps are initially aligned, they will drift off if there is a frequency error.      Normally for asynchronous sources we insert a frame sync, but that does not seem to be expected here.  How will the 100ms limit be maintained if the sources have a 50ppm frequency offset?
